petab/v2/petab1to2.py

```diff
@@ -455,4 +455,21 @@ def update_prior(row):
         errors="ignore",
     )
 
+    # if uniform, we need to explicitly set the parameters
+    def update_prior_pars(row):
+        prior_type = row.get(v2.C.PRIOR_DISTRIBUTION)
+        prior_pars = row.get(v2.C.PRIOR_PARAMETERS)
+
+        if prior_type not in (v2.C.UNIFORM, v2.C.LOG_UNIFORM) or not pd.isna(
+            prior_pars
+        ):
+            return prior_pars
+
+        return (
+            f"{row[v2.C.LOWER_BOUND]}{v2.C.PARAMETER_SEPARATOR}"
+            f"{row[v2.C.UPPER_BOUND]}"
+        )
```

Comment:
Hm, the v1 spec implies that users can optionally provide bounds for `uniform` priors in the `objectivePriorParameters` column. So, should we use those if they are provided?

Reply:
Yes, they will have been renamed to `priorParameter` further up, and are returned just above if `not pd.isna(prior_pars)`.

Reply:
Yes, the double-negative got me :sweat_smile: Could be this, but fine as is

```python
        # apply default value for uniform priors
        if prior_type in (v2.C.UNIFORM, v2.C.LOG_UNIFORM) and pd.isna(
            prior_pars
        ):
            return (
                f"{row[v2.C.LOWER_BOUND]}{v2.C.PARAMETER_SEPARATOR}"
                f"{row[v2.C.UPPER_BOUND]}"
            )

        return prior_pars
```